src/lang/english.txt

```diff
@@ -1520,6 +1520,9 @@ STR_CONFIG_SETTING_MINUTES_PER_YEAR_VALUE                       :{NUM}
 ###setting-zero-is-special
 STR_CONFIG_SETTING_MINUTES_PER_YEAR_FROZEN                      :0 (calendar time frozen)
 
+STR_CONFIG_SETTING_PAYMENT_TIME_SCALE                           :Time scale for cargo payments: {STRING2}
+STR_CONFIG_SETTING_PAYMENT_TIME_SCALE_HELPTEXT                  :Adjusts the time scale factor (in percentage) in the cargo payment calculations. A lower value means faster deliveries, resulting in higher payments.
```

Comment:
```suggestion
STR_CONFIG_SETTING_PAYMENT_TIME_SCALE_HELPTEXT                  :Adjust how quickly cargo loaded on a vehicle decreases in value over time. A higher value means it loses value more quickly, making the game more difficult
```

Reply:
While I somewhat understand the desire for this not to be confused with timekeeping mode you can't just monopolize every time-related word :p If anything, what you did is traditionnally referred to as "day length", and the main setting is "minutes per year". And payment is a function of distance and time, so some reference to time is unavoidable here. At the very least it should be searchable by "time" IMO. As "aging" is just a term specific to newgrf development and is not used in the game iterface at all. And in OpenTTD context in general seems to more often refer to vehicle age than cargo age. It's also not technically correct as aging period has inverse relation with transit time. And I would definitely avoid any reference to difficulty here as whilst somewhat true this is not a setting that should be used to make the game "harder" or "easier".

The graph looks are also just a matter of representation. It could be done by having lines stay constant and changing time axis instead.

On the other hand this should also be searchable by "distance". But that relation is via speed so calling it distance scaling would definitely be misleading. And I'd also like to avoid it being called "time scale" directly even though that's exactly what it is. So far the best I came up with is "Transfer time scale" to at least divert some attention from the "time" here.